recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - aiofiles >=22.1.0,<23
     - aiosqlite >=0.17.0,<1
     - python
-    - y-py >=0.5.3,<0.6.0
+    - y-py >=0.6.0,<0.7.0
 
 test:
   source_files:
```

Comment:
pytest-cov doesn't seem to be needed in tests, the -cov related flags aren't being used.

Reply:
Thanks. I removed it.